### PR TITLE
Olya changes

### PR DESCRIPTION
**Pull request by dchaplinsky**

### CODE REVIEW

File: utils/mappings.py [hunk 1]
```diff
@@ -274,6 +274,193 @@
...
+    ('workPost', r'регулят.*служ', 'Інші державні служби, комісії, і т.п.'),
+    ('workPost', r'експорт.*контр', 'Інші державні служби, комісії, і т.п.'),
+    ('workPost', r'служ.*наркот', 'Інші державні служби, комісії, і т.п.'),
+    ('workPost', r'інт.*влас', 'Інші державні служби, комісії, і т.п.'),
+    ('workPost', r'служ.*ветеран', 'Інші державні служби, комісії, і т.п.'),
+    ('workPost', r'держ.*фін.*інсп', 'Інші державні служби, комісії, і т.п.'),
+    ('workPost', r'гео.*кадас', 'Кабмін, міністерства та підлеглі органи'),
+    ('workPost', r'корупц.*бю', 'НАБУ'),
+    ('workPost', r'набу', 'НАБУ'),
+    ('workPost', r'нац*аг*зап*корупц', 'НАЗК'),
```
Comment (excieve) [inline utils/mappings.py]: This is probably missing `.` in `.*`, same in the originals. #16 mentions it.

File: utils/mappings.py [hunk 2]
```diff
@@ -274,6 +274,193 @@
     ('workPlace', r'держ.*риб.*аг', 'Кабмін, міністерства та підлеглі органи'),
     ('workPlace', r'упр.*охор.*вод.*риб', 'Кабмін, міністерства та підлеглі органи'),
     ('workPlace', r'лісов.*госп', 'Кабмін, міністерства та підлеглі органи'),
+
+    # Sometime declarants are ignorant enough and do write both office and position in
```
Comment (excieve) [inline utils/mappings.py]: Please avoid repeating these as much as possible (it's bad enough as it is already). E.g. first item of the tuple can be another tuple or a list and `pattern_classify()` could be changed to iterate over it as well.

**Comment by dchaplinsky** [comment]
I've made requested changes, testing it now.

Side effects of that tho is that it changes the priority of application of patterns. I.e old version of the code was falling back to workPost only if all workPlace patterns will fail.

**Comment by excieve** [comment]
Feel free to merge when you're happy with testing results.

**Pull request merged**